libsolidity/ast/AST.h

```diff
@@ -362,6 +386,24 @@ class Documented
 	ASTPointer<ASTString> m_documentation;
 };
 
+/**
+ * Abstract class that is added to each AST node that can receive documentation.
+ */
+class NatspecDocumented
```

Comment:
Introducing this class means you also have to update all `accept` functions of all impacted classes to visit the `m_documentation` member, right?

Reply:
Done.